python-sdk/tests/databases/test_snowflake.py

```diff
@@ -94,6 +94,10 @@ def test_load_file_to_table_natively_for_fallback_raises_exception_if_not_enable
         metadata=Metadata(database="SNOWFLAKE_DATABASE", schema="SNOWFLAKE_SCHEMA"),
     )
     database = SnowflakeDatabase()
+    database.load_options = SnowflakeLoadOptions(
+        copy_options={"ON_ERROR": "CONTINUE"},
```

Comment:
I think ON_ERROR=CONTINUE and `validation_mode` are mutually exclusive, if that's the case wouldn't it make sense to keep ON_ERROR=CONTINUE as default? because anyone who has specified the `validation_mode` will anyway run into an issue when loading fails, right?

Reply:
Not necessarily @utkarsharma2.  `validation_mode` is not only for errors. There are three modes: `RETURN_n_ROWS` | `RETURN_ERRORS` | `RETURN_ALL_ERRORS`. And this just validates the data it doesn't load it.

Reply:
Validation mode runs only when specified and it doesn't load the data. It just instructs the COPY command to validate the data files instead of loading them into the specified table; i.e. the COPY command tests the files for errors but does not load them.